src/themes/default/pages/Product.vue

```diff
@@ -143,11 +143,19 @@
                 :name="$t('Quantity')"
                 v-model="product.qty"
                 :min="1"
+                @input="$v.$touch()"
```

Comment:
```suggestion
                @blur="$v.$touch()"
```
it's a good practice to show validation messages after user stops editing it for the first time. And when you once touched field then validation will be instant.

Reply:
of course please ensure, that BaseInputNumber emits blur:)